### PR TITLE
Removed tornado source code, added dependency

### PR DESCRIPTION
**Pull request by MBlistein**
There are debian packages for both tornado and backports, that contain newer versions as the source code copied inside of rosbridge_server. 
Therefore the source code has been deleted and replaced with run dependencies on the corresponding debian packages.

### CODE REVIEW

**Comment by T045T** [comment]
There have been attempts to switch to an external version before (see #162 for an overview and links to related issues).

I'm not sure if there aren't any subtle issues with using an external `tornado`.
@rctoris , @jihoonl , @jonbinney: Looks like you were involved in the previous effort, do you know of any showstoppers here?

File: rosbridge_server/package.xml [hunk 1]
```diff
@@ -15,12 +15,10 @@
...
-  <build_depend>rosbridge_library</build_depend>
-  <build_depend>rosapi</build_depend>
-  <build_depend>rospy</build_depend>
+  <run_depend>python-backports.ssl-match-hostname</run_depend>
+  <run_depend>python-tornado</run_depend>
   <run_depend>python-twisted-core</run_depend>
   <run_depend>rosbridge_library</run_depend>
   <run_depend>rosapi</run_depend>
   <run_depend>rospy</run_depend>
-  <run_depend>rosauth</run_depend>
```
Comment (jihoonl) [inline rosbridge_server/package.xml]: don't remove `rosauth`. It is in the code

Comment (MBlistein) [reply rosbridge_server/package.xml]: Ok, @jspricke will create a debian package and we will look at testing together with @Behery  next week.

**Comment by MBlistein** [comment]
Hey,
big thanks to jspricke who made a debian package and Behery who tested it. It seems to work out:
- when installing / upgrading, it overwrites the existing installation in opt/ros/kinetic
- tornado and backport are uninstalled (but corresponding debian packages installed via apt)
- the rosbridge server is functional:
- [ ]  subscribing/ unsubscribing/ publishing to topics works
- [ ]  service calls go through, advertising a service also works

Should be good to go, I will update the pull request with your remark in a minute.

**Comment by jihoonl** [comment]
Since we plan to squash and merge, it should be fine to have a merge commit. It will get squashed as long as there is no conflict.

**Comment by jihoonl** [comment]
Let me ping ros maintainer to have a last check.

**Comment by MBlistein** [comment]
Ok, thanks! I did rebase in the meantime.

**Comment by jihoonl** [comment]
@MBlistein hey I just released the newer version of rosbridge. You may want to test `ros-shadow-fixed` for final checking.

**Comment by MBlistein** [comment]
@jihoonl Sorry for the late reply. Tested it (thanks to @Behery ) and it seems fine. Luckily, since already released out of shadow-fixed :)

**Pull request merged**